releaseNotes/SpringCloudAzureFeatureManagement.md

```diff
@@ -1,9 +1,16 @@
 # spring-cloud-azure-feature-management
+
 [Source code][source_code] | [Package (Maven)][package] | [Samples][samples] | [Product documentation][docs]
 
 # spring-cloud-azure-feature-management-web
+
 [Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Samples web][samples_web] | [Product documentation][docs]
 
+## 1.1.0.M6 - December 09, 2019
+
+* Added support for '.' in key values from files.
```

Comment:
Not sure what this means... do we support file name to have a dot or do we support the key name to have a dot?